pkg/subscriber/helmrepo/helm_subscriber_item.go

```diff
@@ -288,27 +288,26 @@ func (hrsi *SubscriberItem) manageHelmCR(indexFile *repo.IndexFile, repoURL stri
 			return err
 		}
 
+		dplkey := types.NamespacedName{
+			Name:      dpl.Name,
+			Namespace: dpl.Namespace,
+		}
+		pkgMap[dplkey.Name] = true
+
 		validGvk := hrsi.synchronizer.GetValidatedGVK(helmGvk)
 		err = hrsi.synchronizer.AddTemplates(syncsource, hostkey, []kubesynchronizer.DplUnit{{Dpl: dpl, Gvk: *validGvk}})
 
 		if err != nil {
-			klog.Info("eror in registering :", err)
-			err = utils.SetInClusterPackageStatus(&(hrsi.Subscription.Status), dpl.GetName(), err, nil)
+			klog.Error("error in registering :", err)
 
-			if err != nil {
-				klog.Info("error in setting in cluster package status :", err)
+			if serr := utils.SetInClusterPackageStatus(&(hrsi.Subscription.Status), dpl.GetName(), err, nil); serr != nil {
+				klog.Info("error in setting in cluster package status :", serr)
```

Comment:
should it be `Info` or `Error` ? Or maybe at leastn `Warn` or something similar

Reply:
Yup, forgot to change it